### ISSUE TITLE
[DocDB][LST] tserver crash in yb::(anonymous namespace)::HandleStackTraceSignal

### ISSUE THREAD

**Issue by def-** [yugabyte/yugabyte-db#13605]
Jira Link: [DB-3180](https://yugabyte.atlassian.net/browse/DB-3180)
### Description

There was a segfault on yb-tserver:
```
$ cd ~/code/yugabyte-db
$ git checkout ff45d27295811c9e0bf411345095a7f504aa9c9c
$ ./yb_build.sh release
$ bin/yb-ctl --replication_factor 3 create --tserver_flags=ysql_enable_packed_row=true,yb_enable_read_committed_isolation=true,ysql_num_shards_per_tserver=2,enable_stream_compression=true,stream_compression_algo=2,yb_num_shards_per_tserver=2 --master_flags=yb_enable_read_committed_isolation=true,enable_stream_compression=true,stream_compression_algo=2,enable_automatic_tablet_splitting=true,tablet_split_low_phase_shard_count_per_node=1,tablet_split_high_phase_shard_count_per_node=5,ysql_enable_packed_row=true
$ cd ~/code/yb-long-system-test
$ git checkout 4124bba373813f00a78d9aae54171c7116016802
$ while true; do python3.9 ./long_system_test.py --nodes=127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 --threads=10 --runtime=60 --max-columns=15 --complexity=full; sleep 10; done
[...]
2022-08-12 02:39:14,995 MainThread INFO     Reproduce with: git checkout 4124bba3 && ./long_system_test.py --nodes=127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 --threads=10 --runtime=60 --max-columns=15 --complexity=full --seed=187607
2022-08-12 02:39:15,035 MainThread INFO     Database version: PostgreSQL 11.2-YB-2.15.3.0-b0 on x86_64-pc-linux-gnu, compiled by clang version 13.0.1 (https://github.com/yugabyte/llvm-project.git 191e3a05a55c8671bcc88d7387c04c55a4310500), 64-bit
2022-08-12 02:39:15,038 MainThread INFO     Creating tables for database db_lst_187607
2022-08-12 02:39:47,043 MainThread INFO     Starting worker_0: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-12 02:39:47,054 MainThread INFO     Starting worker_1: RandomSelectAction, SetConfigAction
2022-08-12 02:39:47,062 MainThread INFO     Starting worker_2: RandomSelectAction, SetConfigAction
2022-08-12 02:39:47,070 MainThread INFO     Starting worker_3: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-12 02:39:47,071 MainThread INFO     Starting worker_4: RandomSelectAction, SetConfigAction
2022-08-12 02:39:47,101 MainThread INFO     Starting worker_5: RandomSelectAction, SetConfigAction
2022-08-12 02:39:47,104 MainThread INFO     Starting worker_6: RandomSelectAction, SetConfigAction
2022-08-12 02:39:47,109 MainThread INFO     Starting worker_7: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-12 02:39:47,117 MainThread INFO     Starting worker_8: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-08-12 02:39:47,121 MainThread INFO     Starting worker_9: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-12 02:39:57,132 MainThread INFO     Worker queries/s: [002.4][000.9][000.5][001.8][001.0][000.8][000.3][001.9][001.1][002.8]
2022-08-12 02:40:07,146 MainThread INFO     Worker queries/s: [002.7][000.8][000.6][001.8][001.1][001.1][000.3][002.4][001.3][001.9]
2022-08-12 02:40:17,156 MainThread INFO     Worker queries/s: [002.0][000.9][000.7][001.9][001.0][000.4][000.3][003.2][000.4][002.8]
2022-08-12 02:40:27,169 MainThread INFO     Worker queries/s: [003.0][001.6][001.3][004.3][003.6][001.0][001.2][003.3][000.6][005.7]
2022-08-12 02:40:37,179 MainThread INFO     Worker queries/s: [004.1][001.4][000.7][001.8][001.1][001.1][000.8][002.5][001.0][004.6]
2022-08-12 02:40:47,189 MainThread INFO     Worker queries/s: [001.9][000.7][000.7][001.5][001.3][000.6][000.5][002.4][001.2][001.5]
2022-08-12 02:40:57,201 MainThread INFO     Worker queries/s: [001.4][000.9][000.3][000.8][000.8][000.4][000.2][001.9][001.9][001.2]
2022-08-12 02:41:07,218 MainThread INFO     Worker queries/s: [001.1][000.6][000.3][001.3][000.5][000.7][000.3][001.2][001.2][001.3]
2022-08-12 02:41:17,231 MainThread INFO     Worker queries/s: [001.6][000.8][000.4][001.3][000.9][000.6][000.4][002.0][001.6][000.8]
2022-08-12 02:41:27,241 MainThread INFO     Worker queries/s: [001.4][000.9][000.3][001.4][000.7][000.6][000.4][001.7][001.7][000.0]
2022-08-12 02:41:37,251 MainThread INFO     Worker queries/s: [001.8][000.4][000.6][001.8][001.1][000.6][000.3][002.3][001.4][001.8]
2022-08-12 02:41:47,265 MainThread INFO     Worker queries/s: [003.6][001.1][000.5][004.2][001.0][000.5][000.6][003.3][000.8][002.7]
2022-08-12 02:41:57,275 MainThread INFO     Worker queries/s: [002.1][000.9][000.4][001.7][000.8][000.6][000.4][001.8][001.6][002.1]
2022-08-12 02:42:07,285 MainThread INFO     Worker queries/s: [001.4][000.3][000.2][001.7][001.1][000.6][000.4][002.4][001.2][001.3]
2022-08-12 02:42:17,295 MainThread INFO     Worker queries/s: [000.7][000.8][000.3][001.3][001.2][000.7][000.3][001.0][001.7][001.4]
2022-08-12 02:42:27,305 MainThread INFO     Worker queries/s: [002.5][000.9][000.4][001.6][000.9][000.6][000.4][001.3][001.2][001.7]
2022-08-12 02:42:37,315 MainThread INFO     Worker queries/s: [001.6][000.7][000.2][002.3][001.1][000.6][000.2][001.6][001.3][001.7]
2022-08-12 02:42:47,325 MainThread INFO     Worker queries/s: [000.2][000.7][000.3][001.5][000.8][000.8][000.1][001.6][001.5][001.4]
2022-08-12 02:42:57,336 MainThread INFO     Worker queries/s: [000.8][000.6][000.3][002.0][000.8][000.9][000.3][002.1][001.1][002.0]
2022-08-12 02:43:07,346 MainThread INFO     Worker queries/s: [000.0][001.0][000.5][001.7][000.9][000.4][000.3][001.4][001.5][001.9]
2022-08-12 02:43:17,356 MainThread INFO     Worker queries/s: [000.0][001.5][000.9][005.1][001.6][000.6][000.8][003.2][000.1][000.4]
2022-08-12 02:43:27,368 MainThread INFO     Worker queries/s: [000.5][000.7][000.8][003.6][001.9][001.0][000.6][003.5][001.0][004.2]
2022-08-12 02:43:37,378 MainThread INFO     Worker queries/s: [002.7][000.4][000.4][001.5][000.6][000.5][000.2][003.4][001.3][002.3]
2022-08-12 02:43:47,388 MainThread INFO     Worker queries/s: [001.8][000.7][000.3][001.5][000.9][000.6][000.1][001.1][001.1][001.3]
2022-08-12 02:43:57,398 MainThread INFO     Worker queries/s: [001.5][001.1][000.6][001.3][001.1][000.6][000.3][001.5][001.5][001.2]
2022-08-12 02:44:07,408 MainThread INFO     Worker queries/s: [001.6][000.7][000.2][001.7][000.8][000.7][000.4][002.2][001.4][001.4]
2022-08-12 02:44:17,418 MainThread INFO     Worker queries/s: [001.7][001.0][000.3][001.8][000.7][000.6][000.5][004.2][000.7][002.2]
2022-08-12 02:44:27,430 MainThread INFO     Worker queries/s: [002.3][000.9][000.5][002.8][001.0][000.3][000.5][003.0][000.8][003.8]
2022-08-12 02:44:37,440 MainThread INFO     Worker queries/s: [002.0][000.8][000.4][001.7][000.6][001.2][000.8][003.1][000.7][004.7]
2022-08-12 02:44:47,450 MainThread INFO     Worker queries/s: [001.1][000.7][000.1][001.1][000.6][000.5][000.3][001.1][001.5][001.4]
2022-08-12 02:44:57,461 MainThread INFO     Worker queries/s: [001.2][001.0][000.3][000.7][000.6][000.6][000.1][001.7][001.6][001.4]
2022-08-12 02:45:07,471 MainThread INFO     Worker queries/s: [001.3][000.7][000.4][001.9][000.8][000.5][000.1][002.2][001.5][001.4]
2022-08-12 02:45:17,482 MainThread INFO     Worker queries/s: [001.5][000.6][000.3][001.0][000.7][000.7][000.1][001.1][000.9][002.1]
2022-08-12 02:45:27,492 MainThread INFO     Worker queries/s: [002.0][000.6][000.3][002.0][001.2][001.0][000.5][003.3][001.2][002.8]
2022-08-12 02:45:37,502 MainThread INFO     Worker queries/s: [001.8][001.0][000.4][001.1][001.1][000.9][000.1][001.3][001.2][002.6]
2022-08-12 02:45:47,512 MainThread INFO     Worker queries/s: [003.1][000.8][000.5][004.0][001.2][000.6][000.9][002.7][000.9][003.0]
2022-08-12 02:45:57,523 MainThread INFO     Worker queries/s: [002.3][000.5][000.2][001.3][000.7][000.5][000.2][002.3][001.3][002.0]
2022-08-12 02:46:07,534 MainThread INFO     Worker queries/s: [001.7][001.1][000.2][001.1][000.9][000.4][000.2][002.0][001.3][001.2]
2022-08-12 02:46:17,544 MainThread INFO     Worker queries/s: [002.0][001.0][000.3][001.1][000.5][000.5][000.3][001.8][001.3][001.9]
2022-08-12 02:46:27,554 MainThread INFO     Worker queries/s: [004.2][001.3][000.2][002.6][000.9][000.9][000.9][002.6][000.6][005.2]
2022-08-12 02:46:37,564 MainThread INFO     Worker queries/s: [005.6][000.8][000.3][002.8][001.0][000.7][000.6][003.4][000.6][003.7]
2022-08-12 02:46:47,575 MainThread INFO     Worker queries/s: [001.1][000.6][000.1][001.9][000.7][000.4][000.2][001.5][001.5][001.0]
2022-08-12 02:46:57,585 MainThread INFO     Worker queries/s: [002.8][000.7][000.6][001.8][001.4][000.8][000.6][003.2][000.6][003.5]
2022-08-12 02:47:07,595 MainThread INFO     Worker queries/s: [002.3][001.1][000.1][001.2][000.5][000.4][000.2][001.6][000.8][002.8]
2022-08-12 02:47:17,605 MainThread INFO     Worker queries/s: [002.8][000.9][000.3][001.7][000.6][000.8][000.5][002.8][000.8][002.3]
2022-08-12 02:47:27,615 MainThread INFO     Worker queries/s: [002.3][001.5][000.7][002.5][000.9][000.7][000.3][000.8][001.0][002.1]
2022-08-12 02:47:37,625 MainThread INFO     Worker queries/s: [002.4][000.7][000.4][001.5][000.6][000.6][000.1][001.7][001.3][001.5]
2022-08-12 02:47:47,635 MainThread INFO     Worker queries/s: [001.3][000.7][000.4][001.7][000.5][000.8][000.4][001.1][001.4][001.0]
2022-08-12 02:47:57,646 MainThread INFO     Worker queries/s: [001.3][000.7][000.4][001.1][000.9][000.5][000.3][001.7][001.2][001.3]
2022-08-12 02:48:07,658 MainThread INFO     Worker queries/s: [002.1][000.8][000.3][002.0][000.8][000.6][000.2][002.4][000.7][001.8]
2022-08-12 02:48:17,668 MainThread INFO     Worker queries/s: [003.4][001.7][001.1][003.3][000.9][001.3][000.6][003.3][000.3][004.8]
2022-08-12 02:48:27,676 MainThread INFO     Worker queries/s: [000.0][000.0][000.0][000.0][000.0][000.0][000.6][000.0][000.0][000.2]
2022-08-12 02:48:37,685 MainThread INFO     Worker queries/s: [000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0]
2022-08-12 02:48:47,692 MainThread INFO     Worker queries/s: [000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0]
2022-08-12 02:48:57,703 MainThread INFO     Worker queries/s: [000.5][001.2][000.3][000.5][000.3][000.1][001.2][001.7][000.1][000.5]
2022-08-12 02:49:05,012 worker_1   ERROR    Unexpected query failure: InternalError_
Query: SELECT '(-26.586273217906324,36.348008007543484]'::NUMRANGE, '[-55,50)'::INT4RANGE, '1992-12-02', CAST((CAST((-8) AS FLOAT8)) AS BIGINT), -37 FROM tg4_0 ORDER BY 2 DESC LIMIT 3;
  values: None
  runtime: 2022-08-12 02:48:56.259 - 2022-08-12 02:49:05.012
  supports explain: True
  supports rollback: True
  affected rows: None
Action: RandomSelectAction
Error class: InternalError_
Error code: XX000
Error message: ERROR:  Query error: GetTransactionStatus RPC (request call id 29660189) to 127.0.0.1:9100 timed out after 5.000s
CONTEXT:  Catalog Version Mismatch: A DDL occurred while processing this query. Try again.
Transaction isolation level: default
DB Node: host: 127.0.0.2, port: 5433
DB Backend PID: 1306839
```
```
[dfelsing@devp yugabyte-db]$ coredumpctl gdb 1125207
           PID: 1125207 (yb-tserver)
           UID: 1000 (dfelsing)
           GID: 1000 (dfelsing)
        Signal: 11 (SEGV)
     Timestamp: Fri 2022-08-12 02:48:59 UTC (3h 18min ago)
  Command Line: /home/dfelsing/code/yugabyte-db/build/latest/bin/yb-tserver --fs_data_dirs /home/dfelsing/yugabyte-data/node-1/disk-1 --webserver_interface 127.0.0.1 --rpc_bind_addresses 127.0.0.1 --v 0 --version_file_json_path=/home/dfelsing/code/yugabyte-db/build/release-clang13-linuxbrew-dynamic --tserver_master_addrs=127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 --yb_num_shards_per_tserver=2 --redis_proxy_bind_address=127.0.0.1:6379 --cql_proxy_bind_address=127.0.0.1:9042 --local_ip_for_outbound_sockets=127.0.0.1 --use_cassandra_authentication=false --ysql_num_shards_per_tserver=2 --default_memory_limit_to_ram_ratio=0.65 --enable_ysql=true --pgsql_proxy_bind_address=127.0.0.1:5433 --ysql_enable_packed_row=true --yb_enable_read_committed_isolation=true --ysql_num_shards_per_tserver=2 --enable_stream_compression=true --stream_compression_algo=2 --yb_num_shards_per_tserver=2
    Executable: /home/dfelsing/code/yugabyte-db/build/release-clang13-linuxbrew-dynamic/bin/yb-tserver
 Control Group: /
         Slice: -.slice
       Boot ID: 28c3202246b74d4faa22a31eed01e6af
    Machine ID: 0be2e9f74450408493dd574dc205a4c9
      Hostname: devp
       Storage: /var/lib/systemd/coredump/core.yb-tserver.1000.28c3202246b74d4faa22a31eed01e6af.1125207.1660272539000000.lz4
       Message: Process 1125207 (yb-tserver) of user 1000 dumped core.

                Stack trace of thread 1129975:
                #0  0x00007fb06043acd9 n/a (libgcc_s.so.1)
                #1  0x00007fb06043c5f8 _Unwind_Backtrace (libgcc_s.so.1)
                #2  0x00007fb0626242bf __GI___backtrace (libc.so.6)
                #3  0x00007fb0641cbdf4 n/a (/home/dfelsing/code/yugabyte-db/build/release-clang13-linuxbrew-dynamic/lib/libyb_util.so)
                #4  0x00007fb06428dac8 n/a (/home/dfelsing/code/yugabyte-db/build/release-clang13-linuxbrew-dynamic/lib/libyb_util.so)
                #5  0x00007fb062564120 __restore_rt (libc.so.6)
                #6  0x00007fb0628dcb09 __lll_lock_wait_private (libpthread.so.0)
                #7  0x00007fb0628d5ab1 get_cached_stack (libpthread.so.0)
                #8  0x000000013d611d70 n/a (n/a)
(gdb) bt
#0  0x00007fb06043acd9 in ?? () from /opt/yb-build/brew/linuxbrew-20181203T161736v9/lib/libgcc_s.so.1
#1  0x00007fb06043c5f8 in _Unwind_Backtrace ()
   from /opt/yb-build/brew/linuxbrew-20181203T161736v9/lib/libgcc_s.so.1
#2  0x00007fb0626242bf in __GI___backtrace (array=<optimized out>, size=<optimized out>)
    at ../sysdeps/x86_64/backtrace.c:110
#3  0x00007fb0641cbdf4 in yb::StackTrace::Collect (this=0x7faf500d3a48, skip_frames=2)
    at ../../../../../src/yb/util/debug-util.cc:434
#4  0x00007fb06428dac8 in yb::(anonymous namespace)::HandleStackTraceSignal (signum=<optimized out>)
    at ../../../../../src/yb/util/stack_trace.cc:183
#5  <signal handler called>
#6  __lll_lock_wait_private () at ../sysdeps/unix/sysv/linux/x86_64/lowlevellock.S:93
#7  0x00007fb0628d5ab1 in get_cached_stack (memp=<synthetic pointer>, sizep=<synthetic pointer>)
    at allocatestack.c:181
#8  allocate_stack (stack=<synthetic pointer>, pdp=<synthetic pointer>, attr=0x7faf500d41a0)
    at allocatestack.c:492
#9  __pthread_create_2_1 (newthread=0xe991d48, attr=<optimized out>,
    start_routine=0x7fb0642a0bc0 <yb::Thread::SuperviseThread(void*)>, arg=0xe991d40)
    at pthread_create.c:526
#10 0x000000013d611d70 in ?? ()
#11 0x000000004635f020 in ?? ()
#12 0x0000000000000000 in ?? ()
```
[lst.zip](https://github.com/yugabyte/yugabyte-db/files/9314071/lst.zip)
yugabyte-data directory available from within Yugabyte organization: https://drive.google.com/file/d/1GovFzhJuAaF6817vaGZ2x4dWeCQ1wBsd/view?usp=sharing

**Comment by rthallamko3** [comment]
@def- , It looks like DDLs occurred during the period, that could have caused the dumps. Were you not able to get the backtrace for the above issue, that can help with better routing.

> 
Error message: ERROR:  Query error: GetTransactionStatus RPC (request call id 29660189) to 127.0.0.1:9100 timed out after 5.000s
CONTEXT:  Catalog Version Mismatch: A DDL occurred while processing this query. Try again.

**Comment by def-** [comment]
This crash seems to occur relatively often, got it again yesterday. Does this help?
```
2022-08-16 20:30:12,612 MainThread INFO     Reproduce with: git checkout 4124bba3 && ./long_system_test.py --nodes=127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 --threads=10 --runtime=60 --max-columns=15 --complexity=full --seed=026001
2022-08-16 20:30:14,135 MainThread INFO     Database version: PostgreSQL 11.2-YB-2.15.3.0-b0 on x86_64-pc-linux-gnu, compiled by clang version 13.0.1 (https://github.com/yugabyte/llvm-project.git 191e3a05a55c8671bcc88d7387c04c55a4310500), 64-bit
2022-08-16 20:30:14,138 MainThread INFO     Creating tables for database db_lst_026001
2022-08-16 20:32:36,864 MainThread INFO     Starting worker_0: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-08-16 20:32:36,869 MainThread INFO     Starting worker_1: RandomSelectAction, SetConfigAction
2022-08-16 20:32:36,874 MainThread INFO     Starting worker_2: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-08-16 20:32:36,875 MainThread INFO     Starting worker_3: RandomSelectAction, SetConfigAction
2022-08-16 20:32:36,875 MainThread INFO     Starting worker_4: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-08-16 20:32:36,876 MainThread INFO     Starting worker_5: RandomSelectAction, SetConfigAction
2022-08-16 20:32:36,877 MainThread INFO     Starting worker_6: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-16 20:32:36,879 MainThread INFO     Starting worker_7: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-16 20:32:36,879 MainThread INFO     Starting worker_8: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-16 20:32:36,886 MainThread INFO     Starting worker_9: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-08-16 20:32:44,134 worker_5   ERROR    Unexpected query failure: InternalError_
Query: SELECT coalesce(upper(('(1914-08-21,2020-11-13)'::DATERANGE)), ('1925-04-18')), -79.19138933708001, '{"a": 9, "b": ["0", "1", "2"], "c": true}'::jsonb, '{"a": 5, "b": ["0", "1"], "c": true}'::json, -23.674626631166333, (55.323442131779586) / (-48.83582836337004), ('[-52,41]'::INT8RANGE) * ((range_merge(('(,-88)'::INT8RANGE), ('(1,29)'::INT8RANGE))) * ('(68,77)'::INT8RANGE)), count(tg2_0.c7_json), count(tg2_0.c8_float8) FROM tg2_0 LIMIT 18 OFFSET 11;
  values: None
  runtime: 2022-08-16 20:32:38.502 - 2022-08-16 20:32:44.133
  supports explain: True
  supports rollback: True
  affected rows: None
Action: RandomSelectAction
Error class: InternalError_
Error code: XX000
Error message: ERROR:  Query error: GetTransactionStatus RPC (request call id 42234830) to 127.0.0.2:9100 timed out after 5.000s
    @           0x9b4e78  errmsg
    @           0x9e0665  HandleYBStatusAtErrorLevel
    @           0x56e90f  ybFetchNext
    @           0x6f7148  ybcIterateForeignScan
    @           0x6ebbbf  ForeignNext
    @           0x6ce533  fetch_input_tuple
    @           0x6cd6e1  ExecAgg
    @           0x6dc596  ExecLimit
    @           0x6ba1c3  standard_ExecutorRun
    @     0x7f136eebdae5  pgss_ExecutorRun
    @     0x7f136eeb460a  ybpgm_ExecutorRun
    @           0x856c3b  PortalRunSelect
    @           0x85677a  PortalRun
    @           0x853f56  yb_exec_simple_query_impl
    @           0x8544f6  yb_exec_query_wrapper_one_attempt
    @           0x85106b  PostgresMain
    @           0x7bd16c  BackendRun
    @           0x7bc5c0  ServerLoop
    @           0x7b8b7b  PostmasterMain
    @           0x71935d  PostgresServerProcessMain
    @           0x719802  main
    @     0x7f137373c825  __libc_start_main
    @           0x4c6a39  _start

    @           0x9b4e78  errmsg
    @           0x85287b  YBPrepareCacheRefreshIfNeeded
    @           0x850ab7  PostgresMain
    @           0x7bd16c  BackendRun
    @           0x7bc5c0  ServerLoop
    @           0x7b8b7b  PostmasterMain
    @           0x71935d  PostgresServerProcessMain
    @           0x719802  main
    @     0x7f137373c825  __libc_start_main
    @           0x4c6a39  _start

CONTEXT:  Catalog Version Mismatch: A DDL occurred while processing this query. Try again.
Transaction isolation level: repeatable read
DB Node: host: 127.0.0.3, port: 5433
DB Backend PID: 4117463
```
[lst.zip](https://github.com/yugabyte/yugabyte-db/files/9357615/lst.zip)

**Comment by def-** [comment]
I have only seen it with packed columns on, but I haven't had runs with it disabled and recently because of https://github.com/yugabyte/yugabyte-db/issues/14345 I'm not getting much coverage with LST

**Comment by rthallamko3** [comment]
@yusong-yan , This looks like DUP of https://github.com/yugabyte/yugabyte-db/issues/15220 to me. cc @kripasreenivasan , The fix for #15220 is in master, In case this shows up again on LST, let us know.
